_includes/header.html

```diff
@@ -14,6 +14,9 @@
         </li>
       {% endif %}
     {% endfor %}
+    <li class="nav-item">
+      <a href="https://euruko2017.org/2017/06/21/diversity-program.html">Diversity</a>
```

Comment:
You mean relative paths right? :) I saw absolute and relative URLs mixed in the project, but sure, it makes sense. Pushing in a sec